runtime/doc/term.txt

```diff
@@ -592,6 +592,17 @@ before using the mouse:
 	"g<LeftMouse>"	is "<C-LeftMouse>	(jump to tag under mouse click)
 	"g<RightMouse>" is "<C-RightMouse>	("CTRL-T")
 
+							*bracketed-paste-mode*
+Bracketed paste mode allows terminal emulators to distinguish between typed
+text and pasted text.
```

Comment:
> Actually I left the text short and easy to read, omitting all technical details like escape sequences etc. because an interested reader would look it up anyway (and probably find the mentioned link in the top matches)

Still, I don't see much downside in giving it a passing mention, perhaps like this:

```
Bracketed paste mode allows terminal emulators to distinguish between typed text and pasted text. For more information, see the following link:

    https://cirw.in/blog/bracketed-paste
```